tests/functional/adapter/test_constraints.py

```diff
@@ -316,7 +316,7 @@ def expected_error_messages(self):
             "violate the new NOT NULL constraint",
             "(id > 0) violated by row with values:",  # incremental mats
             "DELTA_VIOLATE_CONSTRAINT_WITH_VALUES",  # incremental mats
-            "NOT NULL constraint violated for column",
+            "NOT NULL constraint violated for col",
```

Comment:
The linux ODBC driver error truncates `column` to `col`. This shortened version covers both platforms since the check is a substring check.